packages/contracts/contracts/optimistic-ethereum/OVM/predeploys/OVM_DeployerWhitelist.sol

```diff
@@ -185,28 +137,10 @@ contract OVM_DeployerWhitelist is iOVM_DeployerWhitelist {
             bool _allowed
         )
     {
-        bool initialized = Lib_Bytes32Utils.toBool(
-            Lib_SafeExecutionManagerWrapper.safeSLOAD(KEY_INITIALIZED)
+        return (
+            initialized == false
```

Comment:
If the contract is not initialized, anyone can deploy?

Reply:
That's correct, that's the behavior at the moment. If this isn't desirable we should make a new issue for it.

Reply:
Ayyy, you did it for me, thanks! https://github.com/ethereum-optimism/optimism/issues/560